src/bitcoin-cli.cpp

```diff
@@ -238,9 +238,17 @@ UniValue CallRPC(const std::string& strMethod, const UniValue& params)
 
     // check if we should use a special wallet endpoint
     std::string endpoint = "/";
-    if (GetArg("-wallet", "") != "") {
+    std::string walletName = GetArg("-wallet", "");
+    if (walletName != "") {
         // always use v1 endpoint if -wallet has been provided
-        endpoint = "/v1/wallet/"+urlEncode(GetArg("-wallet", ""));
```

Comment:
In commit "Add urlencode/decode via libevent2"

Should move these bitcoin-cli changes to prior "Add -wallet endpoint support to bitcoin-cli" commit (which doesn't currently) compile.